boards/arm/efm32hg_slstk3400a/efm32hg_slstk3400a.dts

```diff
@@ -64,3 +64,22 @@
 	location-tx = <GECKO_LOCATION(4) GECKO_PORT_F GECKO_PIN(2)>;
 	status = "okay";
 };
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/latest/guides/dts/index.html#flash-partitions
```

Comment:
I'm not sure we should embed http links in the configuration files. This has been done in the past but shouldn't be necessary any more. It's easy to find relevant information searching for 'partitions' on http://docs.zephyrproject.org. The url links tend to get outdated every dozen months or so.